### PR TITLE
docs(ecosystem): add fastify-web-response

### PR DESCRIPTION
**Pull request by erfanium**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by erfanium** [comment]
ref: https://github.com/fastify/fastify/issues/4512#issuecomment-1405062647
Approved there

**Comment by Uzlopak** [comment]
Just a nitpick: would have called it fastify-whatwg-response.

**Comment by erfanium** [comment]
@Uzlopak it was my first thought. but nodejs seems uses `web` naming instead of `whatwg`. for example ReadableStream class is available in `stream/web` module. (instead of `stream/whatwg`)

File: docs/Guides/Ecosystem.md [hunk 1]
```diff
@@ -590,6 +590,8 @@ section.
   should use.
 - [`fastify-wamp-router`](https://github.com/lependu/fastify-wamp-router) Web
   Application Messaging Protocol router for Fastify.
+- [`fastify-web-response`](https://github.com/erfanium/fastify-web-response) 
+  Enables returning whatwg Response and ReadableStream in routes
```
Comment (jsumners) [inline docs/Guides/Ecosystem.md:594]:
```suggestion
- [`fastify-web-response`](https://github.com/erfanium/fastify-web-response) 
  Enables returning web streams objects `Response` and `ReadableStream` in routes.
```

**Review by jsumners — APPROVED** [review]

**Pull request merged**